### PR TITLE
Testing - Packagingv2

### PR DESCRIPTION
**Pull request by Salamandar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by Salamandar** [comment]
!testme

File: scripts/remove
Comment (alexAubin) [inline scripts/remove:19]: Bunch of remaining "REMOVEME?"

**Review by alexAubin — APPROVED** [review]
LGTM

File: scripts/remove
Comment (Salamandar) [reply scripts/remove:19]: ah yes gotta clean that.

**Comment by Salamandar** [comment]
just waiting for the remove script to be tested by the CI once, then merging.

**Comment by olberger** [comment]
Is there a changelog for this update anywhere (but processing the git log) ? (see #44)

**Comment by olberger** [comment]
OK... and how hard would it be to document that, i.e. having a changlog of release 3.3 which mentions no specific update Mailman-wise, etc. ?

**Comment by alexAubin** [comment]
We can add a `doc/PRE_UPGRADE.md`

**Pull request merged**
